src/test/resources/search3.ini

```diff
@@ -1,26 +1,32 @@
-# Search3 configuration properties
+[metrics]
+port = 1234
+tls.enabled = false
 
-# Which handler factory to use?
-handler_factory=com.cloudant.search3.FDBDirectorySearchHandlerFactory
- 
-# Configure how search3 listens for requests
-listen.port = 8443
+[fdb]
+version = 600
+
+[search]
+handler_factory = com.cloudant.search3.FDBDirectorySearchHandlerFactory
+port = 8443
  
 # TLS config
 tls.enabled = true
 tls.cert_file = cert.pem
 tls.key_file = key.pem
 tls.ca_file = ca.pem
- 
-# The FoundationDB version to use
-fdb.version = 600
-  
+
+# Concurrency of search requests (update, delete, query)
```

Comment:
Awesome work bubbling up this information right to the configuration level, this is really great for performance testing.